README.md

```diff
@@ -28,18 +28,34 @@ Please see the [contributing guidelines](CONTRIBUTING.md).
 This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
 
 #### Sample queries
-We want the explorer to have rich samples for calling all APIs in the Microsoft Graph. If you have ideas for sample queries, you can edit `sample-queries.csv` and then run `npm run import:samples`. This will convert the samples in the CSV file to structured objects in `src/app/gen-queries.ts`.
 
-Some sample queries have ids or other string constants that are different for authenticated users and
-the sample tenant.  These tokens are maintained in [tokens.ts](src/app/tokens.ts) and more documentation for them can be found in [base.ts](src/app/base.ts).
+We want the explorer to have rich samples for calling all APIs in the Microsoft Graph. Choose the most relevant queries that demonstrate your feature. GET samples are the only queries that will work against our demo tenant. There are a few things that you need to check before you can add a sample query:
+- [ ]	Does your sample query use [scopes]( https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-v2-scopes) that are available via the [Azure AD v2.0 authentication]( https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-appmodel-v2-overview) endpoint? You’ll need to confirm this before you can add your sample.
+- [ ]	Does Graph Explorer already have the scopes that support your query? If not, you’ll need to add the scopes. See the [permissions](#permissions) section for how you can add your scopes. 
+- [ ]	Does documentation exist for functionality described in your sample query? [Create](https://github.com/microsoftgraph/microsoft-graph-docs) and publish the documentation before you add your sample.
+- [ ]	Is the sample query functionality available on either the beta or v1.0 Graph endpoints? Confirm that the functionality is available on at least of one of these endpoints before submitting your sample query.
+- [ ]	Are you certain that the functionality in the sample query will not change? We don’t want to have outdated samples, so only add samples when you are confident that the API won’t change.
+- [ ]	Does the Graph metadata properly describe the functionality in the sample query? Confirm that either the [v1.0](https://graph.microsoft.com/v1.0/$metadata) or [beta]( https://graph.microsoft.com/beta/$metadata) metadata describes your functionality.
```

Comment:
Why is this a requirement?

Reply:
@cbales Regarding your question about requirements.

1) The sample entry requires a doc link.
2) Sample queries should reflect functionality that has already been made public. We depend on the v1.0 and beta metadata for autocomplete.
3) I don't want to see beta features that are in flux being added to the GE samples.
4) We use the metadata for autocomplete.